assets/sass/components/_restrictions.scss

```diff
@@ -6,43 +6,43 @@
   }
 }
 
-.visitor-restriction-badge {
+.restriction-tag {
```

Comment:
Renaming this class (and the file) because these restrictions styles are used for visitors **and** prisoners.